framework/src/governance/lib/data-catalog-database.ts

```diff
@@ -190,19 +190,23 @@ export class DataCatalogDatabase extends TrackedConstruct {
       });
 
       const logGroup = `arn:aws:logs:${currentStack.region}:${currentStack.account}:log-group:/aws-glue/crawlers*`;
-      crawlerRole.addToPolicy(new PolicyStatement({
-        effect: Effect.ALLOW,
-        actions: [
-          'logs:CreateLogGroup',
-          'logs:CreateLogStream',
-          'logs:PutLogEvents',
-          'logs:AssociateKmsKey',
-        ],
-        resources: [
-          logGroup,
-          `${logGroup}:*`,
-        ],
-      }));
+
+      if (!props.crawlerRole) {
```

Comment:
Either we attach the policy also when the role is provided or we document the permissions required for the role. I vote for 1.